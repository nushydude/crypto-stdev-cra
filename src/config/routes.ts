export const routes = {
  SINGLE: '/single',
  BEST_DCA: '/best-dca',
  BEST_BUY: '/best-buy',
  SETTINGS: '/settings',
  LOGIN: '/login',
  SIGNUP: '/signup',
  PROFILE: '/profile',
  // ----
  PLAYGROUND: '/playground',
};
