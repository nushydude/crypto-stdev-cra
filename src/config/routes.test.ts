// match snapshot
import { routes } from './routes';

describe('routes', () => {
  it('should match snapshot', () => {
    expect(routes).toMatchSnapshot();
  });
});