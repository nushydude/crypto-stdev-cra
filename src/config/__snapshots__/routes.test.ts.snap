// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes should match snapshot 1`] = `
Object {
  "BEST_BUY": "/best-buy",
  "BEST_DCA": "/best-dca",
  "CFAGI": "/cfagi",
  "LOGIN": "/login",
  "PLAYGROUND": "/playground",
  "PROFILE": "/profile",
  "SETTINGS": "/settings",
  "SIGNUP": "/signup",
  "SINGLE": "/single",
}
`;
